src/openzaak/components/besluiten/models.py

```diff
@@ -43,42 +45,73 @@ class Besluit(AuditTrailMixin, APIMixin, models.Model):
         "organisatie die het besluit heeft vastgesteld.",
         db_index=True,
     )
-
-    _besluittype_url = models.URLField(
-        _("extern besluittype"),
+    _besluittype_base_url = models.ForeignKey(
+        Service,
+        null=True,
         blank=True,
-        max_length=1000,
-        help_text=_(
-            "URL-referentie naar extern BESLUITTYPE (in een andere Catalogi API)."
-        ),
+        on_delete=models.PROTECT,
+        related_name="besluit_besluittypen",
+        help_text="Basis deel van URL-referentie naar het extern BESLUITTYPE (in een andere Catalogi API).",
+    )
+    _besluittype_relative_url = models.CharField(
+        _("besluittype relative url"),
+        max_length=200,
```

Comment:
this seems to short, given that the original URL field was able to store 1000 chars. `zgw_consumers.Service.api_root` can store 255 chars, with these added 200 chars we have maximum URL length of 455 chars.